CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Set MSRV for `microbit` and `microbit-v2` crates.
 - Bumped MSRV to 1.79.0.
 
+## [0.15.1] - 2024-07-30
```

Comment:
This should include the items under the 'Unreleased' heading above.

Reply:
Good catch. Fixed, thanks much.